pep-0659.rst

```diff
@@ -1,12 +1,19 @@
 PEP: 659
 Title: Specializing Adaptive Interpreter
 Author: Mark Shannon <mark@hotpy.org>
-Status: Draft
+Status: Withdrawn
 Type: Informational
 Content-Type: text/x-rst
 Created: 13-Apr-2021
 Post-History: 11-May-2021
 
+Withdrawal Notice
+=================
+
+`Per the Steering Council <https://github.com/python/steering-council/issues/180>`__:
+
+    We talked this over and we would like you to please withdraw the PEP since,
```

Comment:
```suggestion
    We talked this over and we would like you to withdraw the PEP since,
```

Reply:
That's a direct quote from the linked issue so I think it's best to remain true to Brett's words.